spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -218,45 +218,24 @@ def all_builds_for_train(train: nil)
         end
       end
 
-      # @return [Array]A list of binaries which are not even yet processing based on the version
-      #   These are all build that have no information except the upload date
-      #   Those builds can also be the builds that are stuck on iTC.
-      def pre_processing_builds
```

Comment:
Very nice catch. This was probably from the old times where processing worked differently on iTunes Connect